.travis.yml

```diff
@@ -36,10 +36,12 @@ matrix:
       depth: 200
     script: npm run screenshot:test -- --no-fetch
     before_install:
-      # Source the script to run it in the same shell process. This ensures that any environment variables set by the
-      # script are visible to subsequent Travis CLI commands.
-      # https://superuser.com/a/176788/62792
-      - source test/screenshot/infra/commands/travis.sh
+    # Source the scripts to export their env vars. See https://superuser.com/a/176788/62792
+    - source scripts/skip-unnecessary-travis-tests.sh
+    - source test/screenshot/infra/commands/travis.sh
     install:
-      - npm install
-      #- npm ls # Noisy output, but useful for debugging npm package dependency version issues
+    - npm install
+    #- npm ls # Noisy output, but useful for debugging npm package dependency version issues
+before_install:
```

Comment:
Is this really supposed to be down here like this? Is this supposed to take effect globally? If so, does it still need to be above for screenshot tests?

Reply:
Yes, this is global, and I already added it for screenshot tests above (because the screenshot config overrides `before_install`) 😀